File: UF1/Practica 4 Documentación/estudiante.js
/**
 * Clase que representa a un estudiante, que es una persona y tiene una universidad asociada.
 * @class
 * @author Xiaobin Zhou
 */
const Persona = require('./Persona');

class Estudiante extends Persona {
    /**
     * Constructor de la clase Estudiante.
     * @constructor
     * @param {string} nombre - El nombre del estudiante.
     * @param {number} edad - La edad del estudiante.
     * @param {string} universidad - La universidad a la que asiste el estudiante.
     */
    constructor(nombre, edad, universidad) {
        super(nombre, edad);
        this.universidad = universidad;
    }

    /**
     * Permite al estudiante indicar que está estudiando.
     */
    estudiar() {
        console.log(`Estoy estudiando en la universidad ${this.universidad}`);
    }

    /**
     * Permite al estudiante indicar que está realizando un examen.
     */
    realizarExamen() {
        console.log(`Estoy realizando un examen en la universidad ${this.universidad}`);
    }
}

module.exports = Estudiante;

File: UF4/Practica 3 JavaDoc/elipse-workpage/src/Estudiante.java

/**
 * Esta clase representa a un estudiante, que es una persona y tiene una universidad asociada.
 * @author Xiaobin Zhou
 */
public class Estudiante extends Persona {
    private String universidad;

    /**
     * Constructor de la clase Estudiante.
     * 
     * @param nombre El nombre del estudiante.
     * @param edad La edad del estudiante.
     * @param universidad La universidad a la que asiste el estudiante.
     */
    public Estudiante(String nombre, int edad, String universidad) {
        super(nombre, edad);
        this.universidad = universidad;
    }

    /**
     * Permite al estudiante indicar que está estudiando.
     */
    public void estudiar() {
        System.out.println("Estoy estudiando en la universidad " + universidad);
    }

    /**
     * Permite al estudiante indicar que está realizando un examen.
     */
    public void realizarExamen() {
        System.out.println("Estoy realizando un examen en la universidad " + universidad);
    }
}